platform/android/export/export.cpp

```diff
@@ -2291,14 +2344,18 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 		//build project if custom build is enabled
 		String sdk_path = EDITOR_GET("export/android/custom_build_sdk_path");
-
 		ERR_FAIL_COND_V_MSG(sdk_path == "", ERR_UNCONFIGURED, "Android SDK path must be configured in Editor Settings at 'export/android/custom_build_sdk_path'.");
 
-		_update_custom_build_project();
+		_update_custom_build_project(); //alters the build.gradle, android manifest, etc.
+        APKExportData ed;
+        //have to change save_gradle_project_file and save_apk_so to put .so files and
+        //all the assets in the right place inside the gradle project
+        Error err = export_project_files(p_preset, save_gradle_project_file, &ed, save_gradle_project_so); //not quite sure what happens here
```

Comment:
We're not going to want to send a PR with a comment saying "//not quite sure what happens here"